frontend/frontend/urls.py

```diff
@@ -0,0 +1,22 @@
+"""frontend URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/1.9/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  url(r'^$', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  url(r'^$', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.conf.urls import url, include
+    2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
+"""
+from django.conf.urls import include, url
+from django.contrib import admin
+
+urlpatterns = [
+    url(r'^api/', include('api.urls')),
+    url(r'^admin/', admin.site.urls),
```

Comment:
@iprunache the admin is just a pretty frontend on top of the MySQL db. Something like PHPMyAdmin but a bit better :) That works on top of Django entities. So yeah, I'd remove it.